app/src/sharedTest/java/org/oppia/app/topic/play/TopicPlayFragmentTest.kt

```diff
@@ -25,13 +29,40 @@ import javax.inject.Singleton
 @RunWith(AndroidJUnit4::class)
 class TopicPlayFragmentTest {
 
+  // TODO(#137): Add following test-cases once story-progress function is implemented and expandable list is introduced.
+  //  Story progress is is displayed correctly.
```

Comment:
Super nit: it might be easier to read this list if it's style as a markdown-esque bullet list, e.g.:

```kotlin
//  - Story progress is displayed correctly
//  - Click on arrow to show and hide expandable list is working correctly
...
```

(also, small typo here with the "is is")

Reply:
Done.